apps/frontend/src/components/cards/controltable/ControlRowDetails.vue

```diff
@@ -16,7 +16,8 @@
                 <v-divider />
                 <br />
               </div>
-              {{ main_desc }}
+              <!-- eslint-disable-next-line -->
```

Comment:
Same as other comments, it would be better to just disable the single rule that is being triggered.